.gitignore

```diff
@@ -5,7 +5,6 @@
 node_modules
 tests/*/dist
 tests/*/package-lock.json
-tests/*/yarn.lock
 yarn-error.log
```

Comment:
@avaly We could remove all yarn assets from gitignore:
* yarn-error.log
* .pnp.*
* .yarn/*
* !.yarn/patches
* !.yarn/plugins
* !.yarn/releases
* !.yarn/sdks
* !.yarn/versions